staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -2653,11 +2658,7 @@ func (i *IngressDescriber) describeIngressV1(ing *networkingv1.Ingress, events *
 		w.Write(LEVEL_0, "Ingress Class:\t%v\n", ingressClassName)
 		def := ing.Spec.DefaultBackend
 		ns := ing.Namespace
-		if def == nil {
```

Comment:
Don't we do the same thing already in here?. Sorry, but I don't understand what has been fixed/changed with this PR.

Reply:
Doing this check is not enough, when `def == nil`, and there is no any `ing.Spec.Rules`, in https://github.com/kubernetes/kubernetes/blob/62e9df085a82a5e49c435e5b66f988095a52fe3b/staging/src/k8s.io/kubectl/pkg/describe/describe.go#L2683 , it will dereference `nil` inside `i.describeBackendV1(ns, def)`.